web-api/src/getConnection.ts

```diff
@@ -1,38 +1,73 @@
-import { CamelCasePlugin, Kysely, PostgresDialect } from 'kysely';
```

Comment:
The basic idea of the changes in this file:

1. We only ever want one main connection per lambda, so why are we handling token refresh logic at all? Offload that to the pool since the pool config accepts `async () => Promise<string>`. When the pool connects, it will get a fresh token. Otherwise, we do not need to reconnect.
2. Implement a health check before running a query. If we do not have an expiration mechanism, we need some way to say, "The connection failed, so re-connect." My research suggests that a connection from Lambda to Aurora in the same AZ should be very fast, so I don't think this would measurably impact performance.